contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1077,7 +1077,11 @@ select
     WHEN 'default' THEN default_collation_name
     ELSE  c.collname
     END as collation_name
-  , case when typnotnull then 0 else 1 end as is_nullable
+  , CASE ti.tsql_type_name
+      WHEN 'timestamp' THEN 0
+      WHEN 'sysname' THEN 0
+    ELSE case when typnotnull then 0 else 1 end
```

Comment:
To summarise So we were getting typnotnull as 0 for these types and now we want to only display these datatypes as not nullable in this view. I think this change doesnt make sense to me since we our displaying a behaviour of the datatype which is different to the created type.
We need to make change in the domain itself and not here.

Reply:
We are falsely displaying information of a datatype for a behaviour which is not supported

Reply:
I tried making change in domain like:-

- CREATE DOMAIN sys.SYSNAME AS sys.VARCHAR(128) NOT NULL;
- CREATE DOMAIN sys.TIMESTAMP AS sys.ROWVERSION NOT NULL;

But this will cause error everywhere where we have assigned null to sysname/timestamp datatype . Eg:-
For sys.systypes view for this line :-
`, CAST(collation_name as sys.sysname) as collation`

we are assigning NULL here and it will throw error as :- `domain sysname does not allow null values`

Reply:
I'm not convinced either, repeating this again to point out the gravity of your change - WE ARE DISPLAYING a feature of the datatype we do not support

Reply:
Ok, then we need to think on this.